### ISSUE TITLE
Evaluation of assembly parameters for different genome/chloroplast ratios

### ISSUE THREAD

**Issue by greatfireball** [chloroExtractorTeam/chloroplast_landscape#4]
Do we need to test all candidate programs with different ratio for known chloroplast/genomic reads? Therefore, we could use a read simulator and species with known chloroplast and genome sequence. Different ratios should be used. This will test each program for its suitability to assembly whole chloroplasts from a read mixture.

We already did this analysis for an old version of chloroExtractor, but  not using the current version.

Opinions?

**Comment by greatfireball** [comment]
Since nobody added a comment to that issue, I discussed that issue with @iimog directly. We decide to evaluate the tools using different simulated data sets. That should be done in parallel to the analysis of real word data sets.
I created a new issue #6. Further comments there.
